mass_mailing_subscription_email/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Camptocamp (http://www.camptocamp.com).
+# @author Iván Todorovich <ivan.todorovich@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Mass Mailing Subscription Email",
```

Comment:
maybe "mass_mailing_subscription_[notify|notification]" explains the goal better?